WNproject/WNapp/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
+     
+    name = models.CharField(max_length=50, verbose_name='Name')
+    email = models.EmailField(max_length=254, verbose_name='Email')
+    password = models.CharField(max_length=32, verbose_name='Password')
+   
+    class Meta:
+        verbose_name = "User"
+        verbose_name_plural = "Users"
+        ordering = ['name']
+
+
+        
+class News(models.Model):
+
+    thumbinail = models.ImageField(upload_to=None, height_field=None, width_field=None, max_length=None, blank=True, null=True)
```

Comment:
Это слово пишется не так =).
```suggestion
    thumbnail = models.ImageField(upload_to=None, height_field=None, width_field=None, max_length=None, blank=True, null=True)
```

Reply:
Упс.